src/main/resources/com/mathworks/ci/MatlabScriptBuilder/config.jelly

```diff
@@ -0,0 +1,8 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    
+	  <f:entry title="MATLAB command" field="matlabCommand">
```

Comment:
Consider shorting this to "Command". This is right under the title "Run MATLAB Command", so it probably does not need to restate "MATLAB".

Reply:
Thats a good point, however I feel Command will become very generic. matlabCommand is rather more aligned to the UI elements . and it could be probably easier to co related the WebElements in general as well as for automation needs in future.

Reply:
I think it is fine if the underlying field name stays "matlabCommand". I'm more interested in the "MATLAB command" title that I think would be better as "Command".

![74829958-24dda180-5338-11ea-8555-1bc03ca9f830](https://user-images.githubusercontent.com/34887852/74938290-b8a48180-53bb-11ea-88c3-9cdcb6fcb78a.png)

Perhaps others can weigh in.

Reply:
I am in favor of Run MATLAB Command-->Command.

Reply:
Done.